examples/employee-app/src/main/resources/DbCreds.properties

```diff
@@ -0,0 +1,5 @@
+# Oracle DB Credentials

```

Comment:
This should use the proper copyright header.

Reply:
Fixed.